python_cross_site_scripting/Banhmipatee11_DES/1.py

```
@@ -0,0 +1,45 @@
+from flask import Flask, render_template, request, send_file, redirect, flash
+import os
+from io import BytesIO
+
+app = Flask(__name__)
+app.secret_key = "your_secret_key"  # dùng để flash thông báo
```

Ruff (0.13.3)

Possible hardcoded password assigned to: "secret_key"

(S105)